spacy/tests/lang/et/test_text.py

```diff
@@ -21,4 +16,11 @@ def test_long_text(et_tokenizer):
 inimõiguste, millest nad sõltuvad, üldine mõistmine ja järgimine;
 """
     tokens = et_tokenizer(text)
-    assert len(tokens) == 122
+    assert len(tokens) == 94
+
+
+@pytest.mark.xfail
+def test_ordinal_number(et_tokenizer):
+    text = "10."
+    tokens = et_tokenizer(text)
+    assert len(tokens) == 1
```

Comment:
Here I think it's important to include at least some context:
```suggestion
    text = "10. detsembril 1948"
    tokens = et_tokenizer(text)
    assert len(tokens) == 3
```